### ISSUE TITLE
Screenshot feature not working

### ISSUE THREAD

**Issue by RogerioRibeiro201505534** [GMOD/jbrowse#1574]
Hi! 

I have been using Jbrowser for the past 3 months (desktop) and for some reason the screenshot feature is not working for me. When I click the "screenshot" it opens up a file explorer as normal and I can save it. However, when I go to that same folder it does not have anything saved. Is this a common occurrence, or might it be a problem with my machine?

Jbrowser version: 1.16.9 
OS: Win10

**Comment by cmdcolin** [comment]
@RogerioRibeiro201505534 hey...thanks for the follow up 

I can't reproruce this on linux, curious if anyone that has windows can try

Also, original thread just for posterity (https://www.biostars.org/p/490779/)

**Comment by cmdcolin** [comment]
of course, can always use the OS print screen for screenshots...the screenshot button was kind of just a fun thing for desktop

in jbrowse 2 we are working on getting high resolutions SVG export capability so in that case the functionality will be more advanced than the screenshot button here https://github.com/GMOD/jbrowse-components/pull/1125
